meta-ostro/conf/distro/ostro.conf

```diff
@@ -236,6 +236,15 @@ PREFERRED_PROVIDER_virtual/java-initial-native = "cacao-initial-native"
 PREFERRED_PROVIDER_virtual/java-native = "cacao-native"
 PREFERRED_PROVIDER_virtual/javac-native = "ecj-bootstrap-native"
 
+# Java runtime: pick the same one as in packagegroup-java-jdk.bb.
+#
+# If we pick something simpler here like openjre-8, then images
+# with components like mraa (which depend on java2-runtime) and
+# packagegroup-java-jdk installed pull in both, which then
+# takes up more disk space (openjdk-8 and openjre-8 share no
+# files).
+PREFERRED_RPROVIDER_java2-runtime = "openjdk-8"
```

Comment:
The package group could be written so that it uses "${PREFERRED_RPROVIDER_java2-runtime}" instead of openjdk-8. If we do that, this comment here needs to be updated.

Shall I make that change?